sql/moz-fx-data-shared-prod/search_terms_derived/search_terms_daily_v1/query.sql

```diff
@@ -0,0 +1,44 @@
+WITH terminal_queries AS (
+  SELECT
+    ARRAY_REVERSE(ARRAY_AGG(sanitized ORDER BY sequence_no, timestamp ASC))[SAFE_OFFSET(0)].*
+  FROM
+    `moz-fx-data-shared-prod.search_terms_derived.merino_log_sanitized_v3` sanitized
+  WHERE
+    DATE(timestamp) > @submission_date - 7
+  GROUP BY
+    sanitized.session_id
+),
+daily_queries AS (
+  SELECT
+    DATE(timestamp) AS submission_date,
+    query,
+    COUNT(*) AS search_sessions
+  FROM
+    terminal_queries
+  GROUP BY
+    1,
+    2
+),
+daily_queries_with_threshold AS (
+  SELECT
+    *,
+    SUM(search_sessions) OVER (
+      PARTITION BY
+        QUERY
+      ORDER BY
+        SUBMISSION_DATE ASC
+      ROWS BETWEEN
+        6 PRECEDING
+        AND CURRENT ROW
```

Comment:
I don't think we need to bound the preceding window here if we're already setting a bound on the `terminal_queries` CTE. 

In fact, I think you can just drop the window clause here. from the docs:
> For aggregate analytic functions, if the ORDER BY clause is present but the window frame clause is not, the following window frame clause is used by default:
> 
> `RANGE BETWEEN UNBOUNDED PRECEDING AND CURRENT ROW`